src/main/scala/com/sourcegraph/sbtsourcegraph/SourcegraphPlugin.scala

```diff
@@ -110,14 +109,16 @@ object SourcegraphPlugin extends AutoPlugin {
       )
       out
     },
-    sourcegraphLsif := {
+    sourcegraphScip := {
       val out = target.in(Sourcegraph).value / "dump.lsif"
       out.getParentFile.mkdirs()
       runProcess(
         sourcegraphCoursierBinary.value ::
           "launch" ::
           "--contrib" ::
-          s"lsif-java:${sourcegraphLsifJavaVersion.value}" ::
+          s"scip-java:${sourcegraphScipJavaVersion.value}" ::
+          "-M" ::
+          "com.sourcegraph.scip_java.ScipJava" ::
```

Comment:
I still had issues here without specifying `-M` which is weird because locally testing I didn't have issues when doing launch. This might be needed as well in the example `sourcegraph.yml`, but I figured we'd try it in CI first before making the change there as well.